packages/console/src/pages/SignInExperience/components/ManageLanguageModal/LanguageEditor.tsx

```diff
@@ -24,10 +26,18 @@ const emptyUiTranslation = createEmptyUiTranslation();
 const LanguageEditor = () => {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
 
-  const { selectedLanguage, setIsDirty, stopAddingLanguage } = useContext(LanguageEditorContext);
+  const { data: signInExperience } = useSWR<SignInExperience, RequestError>('/api/sign-in-exp');
+
+  const { languages, selectedLanguage, setIsDirty, setSelectedLanguage, stopAddingLanguage } =
+    useContext(LanguageEditorContext);
+
+  const [isDeletionAlertOpen, setIsDeletionAlertOpen] = useState(false);
 
   const isBuiltIn = isBuiltInLanguageTag(selectedLanguage);
 
+  const isDefaultLanguage =
+    signInExperience && signInExperience.languageInfo.fallbackLanguage === selectedLanguage;
```

Comment:
nit: can be simplified
```suggestion
  const isDefaultLanguage = signInExperience?.languageInfo.fallbackLanguage === selectedLanguage;
```